website_version_ce/models/website.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-#
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import json
+from openerp import fields, models, api
+from openerp.http import request
+import random
+
+
+class NewWebsite(models.Model):
+
+    _inherit = "website"
+
+    tuto_google_analytics = fields.Boolean("How to get GA key and View ID")
+    google_analytics_view_id = fields.Char('View ID')
+    google_management_authorization = fields.Char('Google authorization')
+
+    @api.model
+    def get_current_version(self, context=None):
+        version = request.env['website_version_ce.version']
+        version_id = request.context.get('version_id')
+
+        if not version_id:
+            request.context['version_id'] = 0
+            return 0, ''
+        return version_id, version.browse(version_id).name
+
+    @api.model
+    def get_current_website(self):
+        website = super(NewWebsite, self).get_current_website()
+        # We just set the cookie for the first visit
+        if 'website_version_ce_experiment' in request.httprequest.cookies:
+            main_experiment = json.loads(request.httprequest.cookies.get(
+                'website_version_ce_experiment'))
+        else:
+            main_experiment = request.context.get(
+                'website_version_ce_experiment', {})
+            exps = self.env["website_version_ce.experiment"].search([
+                ('state', '=', 'running'),
+                ('website_id.id', '=', website.id),
+                ('google_id', 'not in', main_experiment.keys())
+            ])
+            for exp in exps:
+                result = []
+                pond_sum = 0
+                for exp_snap in exp.experiment_version_ids:
+                    result.append([int(exp_snap.frequency)+pond_sum,
+                                   exp_snap.version_id.id])
+                    pond_sum += int(exp_snap.frequency)
+                if pond_sum:
+                    # Setting master default frequency at 50
+                    pond_sum += 50
+                    # Setting cookie to avoid problems when db restarts
+                    main_experiment[exp.google_id] = str(0)
```

Comment:
Type casting a static int to a string seems a bit weird IMO, why not just `"0"`?